website/docs/export.md

```diff
@@ -209,6 +211,90 @@ Dapr: https://dapr.io/
     2023/07/18 20:37:20 main.ExportMsg{ID:"2023-07-18T20:37:19Z", Details:map[string]interface {}{"missing_labels":[]interface {}{"test"}}, EventType:"violation_audited", Group:"constraints.gatekeeper.sh", Version:"v1beta1", Kind:"K8sRequiredLabels", Name:"pod-must-have-test", Namespace:"", Message:"you must provide labels: {\"test\"}", EnforcementAction:"deny", ConstraintAnnotations:map[string]string(nil), ResourceGroup:"", ResourceAPIVersion:"v1", ResourceKind:"Pod", ResourceNamespace:"nginx", ResourceName:"nginx-deployment-58899467f5-j85bs", ResourceLabels:map[string]string{"app":"nginx", "owner":"admin", "pod-template-hash":"58899467f5"}}
     ```
 
+### Quick start with exporting violations on node storage using Disk driver via emptyDir
+
+#### Configure Gatekeeper with Export enabled to Disk
+
+1. Deploy Gatekeeper with disk export configurations.
+
+    ```shell
+    helm upgrade --install gatekeeper gatekeeper/gatekeeper --namespace gatekeeper-system \
```

Comment:
should we move this step _after_ user has updated their values yaml?

Reply:
I am not sure, in quickstart steps user doesnt need to create values.yaml to try it out. So the yaml of variables in `values` is just a guide and not a requirement for users to try this. By that flow I think this is fine 🤔

Reply:
We shouldnt encourage users to use the fake-reader container image in production. IMO we should move this up and then add a warning to replace fake-reader for any production use since its just for demo purpose. WDYT?

Reply:
Updated docs to move up default values before installing Gk with added warning about production use.